Model/Sync/Customers/Main.php

```diff
@@ -97,14 +97,6 @@ public function createCustomerSyncData($response, $magentoCustomerId)
      */
     public function insertOrUpdateCustomerSyncData($customerSyncData)
     {
-        $finalData = [];
-        $finalData[] = [
-            'customer_id'        =>  $customerSyncData['customer_id'],
-            'synced_to_yotpo'    =>  $customerSyncData['synced_to_yotpo'],
-            'response_code'      =>  $customerSyncData['response_code'],
-            'store_id'           =>  $customerSyncData['store_id'],
-            'sync_status'        =>  $customerSyncData['sync_status']
-        ];
-        $this->insertOnDuplicate('yotpo_customers_sync', $finalData);
+        $this->insertOnDuplicate('yotpo_customers_sync', $customerSyncData);
```

Comment:
Are you sure that the data that is being saved in this section is exactly the same as in the original implementation of the method?

Reply:
```suggestion
        $this->insertOnDuplicate('yotpo_customers_sync', [$customerSyncData]);
```

The callers of this method shouldn't be aware of the fact that this should be passed as array.

Reply:
About first comment: 
Yes. its gets hash (array) and recreate the same hash (array) with the same keys.
About second comment:
Why not? insertOnDuplicate can insert array of items, but in this case its 1 item. Also it was passed as array before, it was just called finalData, the commit was taking unneeded recreation of array to shortcut of [$variable].